### DISCUSSION TITLE
[Request] Do you mind sharing the font config

### DISCUSSION THREAD

**Discussion by Darahaas2001** [SpiralLinux/SpiralLinux-project#283] [Q&A]
Hello, I loved the font rendering on spiral Linux even though I know you are using Ubuntu font, when I use it on other Linux distro like PopOS or Endeavour they don't look as polished as spiral Linux, since my requirement needs Ubuntu as a base, I cannot use debian coz of corporate requirements. Do you mind sharing the font config.

Loved your project 🍻

**Comment by Jipok** [comment]
Just copy `/etc/fonts/conf.d` from spiral linux to your `~/.config/fontconfig/conf.d`
